--- websafety.conf	2023-08-12 12:46:52.000000000 +0200
+++ websafety.conf.new	2023-08-17 09:43:57.249370929 +0200
@@ -40,7 +40,7 @@
     ServerAlias example.lan
     ServerAdmin administrator@example.lan
 
-    WSGIDaemonProcess example.lan python-home=/opt/websafety-ui/env python-path=/opt/websafety-ui/var/console user=websafety group=websafety display-name=%{GROUP}
+    WSGIDaemonProcess example.lan python-path=/opt/websafety-ui/var/console user=websafety group=websafety display-name=%{GROUP}
     WSGIProcessGroup  example.lan
     WSGIApplicationGroup %{GLOBAL}
     WSGIScriptAlias / /opt/websafety-ui/var/console/console/wsgi.py
