--- gadgets.cc	2013-07-13 15:25:14.000000000 +0200
+++ gadgets.cc.new	2015-11-09 23:00:47.591124352 +0100
@@ -389,9 +389,9 @@
 
     /*Now sign the request */
     if (properties.signAlgorithm != Ssl::algSignSelf && properties.signWithPkey.get())
-        ret = X509_sign(cert.get(), properties.signWithPkey.get(), EVP_sha1());
+        ret = X509_sign(cert.get(), properties.signWithPkey.get(), EVP_sha256());
     else //else sign with self key (self signed request)
-        ret = X509_sign(cert.get(), pkey.get(), EVP_sha1());
+        ret = X509_sign(cert.get(), pkey.get(), EVP_sha256());
 
     if (!ret)
         return false;
